target/linux/ath79/dts/qca9531_yuncore_cpe830.dts

```diff
@@ -0,0 +1,122 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
```

Comment:
file needs to be renamed to cpe830d

Reply:
no

Your harasmment will lead me to call the police at some point